content/blog/2020/06/2020-06-12-infra-and-aws-donation.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "AWS and Infrastructure Updates - Hello from Infra"
+tags:
+- aws
+- community
+- infrastructure
+- kubernetes
+author: olblak
+opengraph:
+  image: /images/post-images/2020-06-12-infrastructure-opengraph.png
```

Comment:
I do not mind, but I am nit sure what does this image represent

Reply:
I believe it represents the country by country web page access statistics that are listed in the first measurement.  Darker shared represent more distinct visits from that country.  Thus, China, India, and the United States are darker.

Reply:
It needs to be replaced.  I like your idea of creating an OpenGraph image with the sponsor logos.  I'll experiment with it using a Google presentation.

Reply:
> I do not mind, but I am nit sure what does this image represent

This graph is coming from google analytics and represents jenkins.io visitors by country where the darker is 147724

Reply:
Suggest we just add a caption to say what it is rather than redoing a whole new image.